common/smoke-tests/src/main/java/com/azure/KeyVaultSecrets.java

```diff
@@ -47,16 +47,19 @@ public static void main(String[] args) {
         LOGGER.info("IDENTITY - CREDENTIAL");
         LOGGER.info("---------------------");
 
-        /* DefaultAzureCredentialBuilder() is expecting the following environment variables:
+        /* EnvironmentCredentialBuilder() is expecting the following environment variables:
```

Comment:
I would prefer that we keep the test with DAC and then use reflection to set the authority host until this is fixed:  https://github.com/Azure/azure-sdk-for-java/issues/8626

That way the test is consistent across languages: Test DAC and we can remove the reflection once that issue is resolved.  @g2vinay

Reply:
It was implemented in the beta here: https://github.com/Azure/azure-sdk-for-java/blob/5dd0c40303aaf85aac4e7cf36b64a0ab1fe62437/sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredentialBuilder.java#L21

So, ideally this test runs against beta and we use that.

Reply:
Smoke tests run from the latest nightly packages available. Here it is succeeding with the new syntax: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=296102&view=results